src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs

```diff
@@ -73,14 +75,26 @@ private void ShortcutToolbar_Loaded(object sender, RoutedEventArgs e)
             DynamoViewModel.OnRequestShorcutToolbarLoaded(RightMenu.ActualWidth);
         }
 
-        private void SignOutHandler(LoginState status)
+        private void ShortcutToolbar_Unloaded(object sender, RoutedEventArgs e)
```

Comment:
be careful with the unloaded event, it may be called more frequently than you think, you may want to do some null checking in the handler.

Reply:
I see that the unloaded event is not called ever, so putting unsubscribers in Dispose, and calling it on DynamoView disposal.

Reply:
Unloaded is not usually raised by our code but by the wpf framework when a control is unloaded - but the rules are complex.

Reply:
using dispose instead